index.rst

```diff
@@ -2,7 +2,17 @@ Raiden Network Specification
 =================================
 
 
-This is a tentative specification for the `Raiden Network <http://raiden.network/>`_. It's under development and will be further refined in subsequent iterations.
+This is the specification of the `Raiden Network <http://raiden.network/>`_. Its goal is to provide
+an exact description of the Raiden protocol.
+
+For the Raiden client (Raiden node), we will thus describe the format of messages it sends to other
```

Comment:
Let's talk about plural "Raiden clients" instead of "the Raiden client".  The point is then clearer.

Reply:
Good idea.